### PR TITLE
[MPQEditor] Add create menu from OneExplorer

### PR DESCRIPTION
**Pull request by dayo09**
This commit adds the 'create mpq configuration menu' from OneExplorer.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

**Review by stamalakhov — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -283,7 +283,12 @@
       },
       {
         "command": "one.editor.mpq.createFromDefaultExplorer",
-        "title": "Create MPQ json",
+        "title": "ONE: Create Mixed Precision Quantization Configuration",
```
Comment (jyoungyun) [inline package.json:286]: Do we need `ONE:` prefix in title?

Comment (dayo09) [reply package.json:286]: It's the menu shown from the default explorer, so I added it. 
The menu on ONE Explorer doesn't include "ONE:" instead.

Comment (dayo09) [reply package.json:286]: Actually, I am rather worried that ONE Explorer cannot show the created mpq configuration files for now. 😨

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**